content/05-expression.md

```diff
@@ -492,6 +492,43 @@ This syntax is also used to access types within packages in the form of `pack.Ty
 
 The typer ensures that an accessed field actually exist and may apply transformations depending on the nature of the field. If a field access is ambiguous, understanding the [resolution order](type-system-resolution-order) may help.
 
+##### since Haxe 4.3.0
```

Comment:
`####` here? (same line 516)

Reply:
No, the "since Haxe X" header is consistently a level 5 heading in the entire manual.

Reply:
I figured it could be something weird like that x)